modules/core/test/v2/unit/coins/utxo/transaction.ts

```diff
@@ -43,14 +48,18 @@ function run(coin: AbstractUtxoCoin, inputScripts: InputScriptType[]) {
       }
     }
 
-    const unspents = inputScripts.map((type, i) => getUnspent(type, i));
+    function getUnspents(): Unspent[] {
+      return inputScripts.map((type, i) => getUnspent(type, i));
+    }
```

Comment:
this way the error is more readable if it fails

(errors during `before()` show up differently)